src/app/themes.tsx

```diff
@@ -21,6 +21,7 @@ type CustomTheme = {
     accentForegroundColor: string;
     surfaceForegroundColor: string;
     surfaceBackgroundColor: string;
+    disabledColor: string;
```

Comment:
what's disabled colour? is this not just 'normal colour with `<input ... disabled/>`, in which it sets it to some greyed out colour?

Reply:
It was a theme specific grey, but for now I've changed all instances to grey and added a todo message for VFB16